include/OpenColorIO/OpenColorTypes.h

```diff
@@ -576,146 +498,103 @@ enum OptimizationFlags : unsigned long
-//!cpp:function::
 extern OCIOEXPORT LoggingLevel LoggingLevelFromString(const char * s);
 
-//!cpp:function::
 extern OCIOEXPORT const char * TransformDirectionToString(TransformDirection dir);
-//!cpp:function::
 extern OCIOEXPORT TransformDirection TransformDirectionFromString(const char * s);
 
-//!cpp:function::
 extern OCIOEXPORT TransformDirection GetInverseTransformDirection(TransformDirection dir);
-//!cpp:function::
 extern OCIOEXPORT TransformDirection CombineTransformDirections(TransformDirection d1,
                                                                 TransformDirection d2);
 
-//!cpp:function::
 extern OCIOEXPORT const char * ColorSpaceDirectionToString(ColorSpaceDirection dir);
-//!cpp:function::
 extern OCIOEXPORT ColorSpaceDirection ColorSpaceDirectionFromString(const char * s);
 
-//!cpp:function::
 extern OCIOEXPORT const char * BitDepthToString(BitDepth bitDepth);
-//!cpp:function::
 extern OCIOEXPORT BitDepth BitDepthFromString(const char * s);
-//!cpp:function::
 extern OCIOEXPORT bool BitDepthIsFloat(BitDepth bitDepth);
-//!cpp:function::
 extern OCIOEXPORT int BitDepthToInt(BitDepth bitDepth);
 
-//!cpp:function::
 extern OCIOEXPORT const char * AllocationToString(Allocation allocation);
-//!cpp:function::
 extern OCIOEXPORT Allocation AllocationFromString(const char * s);
 
-//!cpp:function::
 extern OCIOEXPORT const char * InterpolationToString(Interpolation interp);
-//!cpp:function::
 extern OCIOEXPORT Interpolation InterpolationFromString(const char * s);
 
-//!cpp:function::
 extern OCIOEXPORT const char * GpuLanguageToString(GpuLanguage language);
-//!cpp:function::
 extern OCIOEXPORT GpuLanguage GpuLanguageFromString(const char * s);
 
-//!cpp:function::
 extern OCIOEXPORT const char * EnvironmentModeToString(EnvironmentMode mode);
-//!cpp:function::
 extern OCIOEXPORT EnvironmentMode EnvironmentModeFromString(const char * s);
 
-//!cpp:function::
 extern OCIOEXPORT const char * CDLStyleToString(CDLStyle style);
-//!cpp:function::
 extern OCIOEXPORT CDLStyle CDLStyleFromString(const char * style);
 
-//!cpp:function::
 extern OCIOEXPORT const char * RangeStyleToString(RangeStyle style);
-//!cpp:function::
 extern OCIOEXPORT RangeStyle RangeStyleFromString(const char * style);
 
-//!cpp:function::
 extern OCIOEXPORT const char * FixedFunctionStyleToString(FixedFunctionStyle style);
-//!cpp:function::
 extern OCIOEXPORT FixedFunctionStyle FixedFunctionStyleFromString(const char * style);
 
-//!cpp:function::
 extern OCIOEXPORT const char * ExposureContrastStyleToString(ExposureContrastStyle style);
-//!cpp:function::
 extern OCIOEXPORT ExposureContrastStyle ExposureContrastStyleFromString(const char * style);
 
-//!cpp:function::
 extern OCIOEXPORT const char * NegativeStyleToString(NegativeStyle style);
-//!cpp:function::
 extern OCIOEXPORT NegativeStyle NegativeStyleFromString(const char * style);
 
-/*!rst::
+// TODO: Move to .rst
```

Comment:
The "move to .rst" comment, is where there is very large exposition about how an API is to be used, rather than what the API does. That sort of long "story" should be a part of the documentation only (.rst) rather than put into the header, unless required IMHO